lib/logger_json/formatters/basic_logger.ex

```diff
@@ -0,0 +1,29 @@
+defmodule LoggerJSON.Formatters.BasicLogger do
+  @moduledoc """
+  Basic JSON log formatter with no vender specific formatting
+  """
+
+  alias LoggerJSON.FormatterUtils
+
+  @behaviour LoggerJSON.Formatter
+
+  @processed_metadata_keys ~w[pid file line function module application]a
+
+  @impl true
+  def format_event(level, msg, ts, md, md_keys) do
+    Map.merge(
+      %{
+        time: FormatterUtils.format_timestamp(ts),
+        severity: Atom.to_string(level),
+        message: IO.iodata_to_binary(msg)
+      },
+      format_metadata(md, md_keys)
+    )
```

Comment:
I think we don't have a requirement to keep metadata on the same level as other entries, I moved it so that we can leverage `json_map` macro and build this part at compile time.

```suggestion
    json_map(
      time: FormatterUtils.format_timestamp(ts),
      severity: Atom.to_string(level),
      message: IO.iodata_to_binary(msg),
      metadata: format_metadata(md, md_keys)
    )
```